sdk/storage/azqueue/internal/generated/autorest.md

```diff
@@ -0,0 +1,70 @@
+# Code Generation - Azure Blob SDK for Golang
+
+### Settings
+
+```yaml
+go: true
+clear-output-folder: false
+version: "^3.0.0"
+license-header: MICROSOFT_MIT_NO_VERSION
+input-file: "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/storage/data-plane/Microsoft.QueueStorage/preview/2018-03-28/queue.json"
```

Comment:
Which STG we have picked up here?

Reply:
There is actually only one STG for Queues.